.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.defaultFormatter": "esbenp.prettier-vscode"
+}
```

Comment:
Please this mask (exclude) this file. Use `git config --global core.exludesfile` setting to add ignore pattern for `.vscode`.